src/core/server/rendering/rendering_service.tsx

```diff
@@ -94,31 +94,18 @@ export class RenderingService {
           defaults: uiSettings.getRegistered(),
           user: includeUserSettings ? await uiSettings.getUserProvided() : {},
         };
-        // Cannot use `uiSettings.get()` since a user might not be authenticated
-        const darkMode =
-          (settings.user?.['theme:darkMode']?.userValue ??
-            uiSettings.getOverrideOrDefault('theme:darkMode')) ||
-          false;
-
-        // At the very least, the schema should define a default theme; the '' will be unreachable
-        const themeVersion =
-          (settings.user?.['theme:version']?.userValue ??
-            uiSettings.getOverrideOrDefault('theme:version')) ||
-          '';
 
         const brandingAssignment = await this.assignBrandingConfig(
-          darkMode,
           opensearchDashboardsConfig as OpenSearchDashboardsConfigType
         );
 
         const metadata: RenderingMetadata = {
           strictCsp: http.csp.strict,
           uiPublicUrl,
           bootstrapScriptUrl: `${basePath}/bootstrap.js`,
+          startupScriptUrl: `${basePath}/startup.js`,
```

Comment:
i feel like we desperately needed this and it allows us to do some cools tuff

Reply:
It's handy, but we definitely want to change how it's built so that we can use our proper TS/JS tooling, rather than writing JS in a big template.